lib/big_query/base/query_result.ex

```diff
@@ -51,6 +51,15 @@ defmodule BigQuery.Base.QueryResult do
           else: String.to_integer(value)
         {:ok, dtim} = DateTime.from_unix(num)
         dtim
+      {"RECORD", %{"v" => v}} -> parse_value(v, "RECORD")
+      {"RECORD", %{"f" => f}} -> parse_value(f, "RECORD")
+      {"RECORD", _} when is_list(value) ->
+        Enum.map(value, &(parse_value(&1, "RECORD")))
+      {"RECORD", _} ->
+        case Integer.parse(value) do
```

Comment:
Nested `RECORD`s are sort of a mess.  (Look at the test to see an example).  So I just keep iterating through the `v`s and `f`s until i hit something that looks like an integer or a string.  There's probably a smarter way to do this ... but this works!